.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+---
+
+name: Continuous integration
+
+on:
+  push:
+
+jobs:
+  main:
+    name: Continuous integration
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      TX_USR: ${{ secrets.TX_USR }}
+      TX_PWD: ${{ secrets.TX_PWD }}
+      TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+      TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Why not using secrets in gopass by using `camptocamp/initialise-gopass-summon-action`?

Reply:
I've open a dedicated issue: https://github.com/camptocamp/c2cgeoform/issues/239